auto_py_to_exe/web/js/configuration.js

```diff
@@ -33,7 +33,8 @@ const getNonPyinstallerConfiguration = () => {
     return {
         outputDirectory: document.getElementById('output-directory').value,
         increaseRecursionLimit: !document.getElementById('recursion-limit-switch').classList.contains('unselected'),
-        manualArguments: document.getElementById('raw-arguments').value
+        manualArguments: document.getElementById('raw-arguments').value,
+        useDarkTheme: document.querySelector("body").hasAttribute("data-dark-theme")
```

Comment:
The configuration is more for the actual data being used to generate the output - we don't need to store theme in here.